lib/parsers/parse5.js

```diff
@@ -30,3 +30,22 @@ exports.render = function (dom) {
     { treeAdapter: htmlparser2Adapter }
   );
 };
+
+// Attempts to match single html elements
+// like <..></..>, <../>, <..>
+var rsingleTag = /^<([a-z][^/\0>:\x20\t\r\n\f]*)[\x20\t\r\n\f]*\/?>(?:<\/\1>)?$/i;
```

Comment:
This could probably be optimized to this (untested):

```js
var rsingleTag = /^<([a-z][^/\0>: \t\r\n\f]*)[ \t\r\n\f]*\/?>(?:<\/\1>)?$/i;
```

Reply:
true \x20 is single space